### ISSUE TITLE
Request For Details

### ISSUE THREAD

**Issue by AlexHu123** [Bingohong/GroupNormalization-tensorflow#1]
Hi, have you trained using deep network or larger dataset? Does group norm go well?
Look forward to your reply!
Thanks in advance!
Alex

**Comment by Bingohong** [comment]
i train u-net. utilize 3 different normalization method:
- batch normalization by keras in-build layers
- group normalization without moving average by other github
- group normalization with moving average by myself

**Comment by AlexHu123** [comment]
Thanks for your reply! How about the results of these methods?

**Comment by Bingohong** [comment]
i upload 3 method result log file in compare_log dir. you can look out these.
however, i think the gn without moving average get a lower val loss about 0.2.
- the gn with moving average keep a high val loss. so i have a question that “does it need to apply moving average in gn?”
- the bn get a val loss about 0.26, higher than gn.

**Comment by AlexHu123** [comment]
Thanks for your reply! Now I try to train on a deep network such as 3D ConvNet in the task of video classification, but get bad results. (the class score are going to be the same, so the network are not working well)

**Comment by Bingohong** [comment]
Why this happen?
